### DISCUSSION TITLE
Is stalebot actually providing more harm than benefit?

### DISCUSSION THREAD

**Discussion by trwnh** [mastodon/mastodon#19211] [General]
I don't think Mastodon gets enough PRs for a significant amount to go stale. And even so, the motivation for the PR may still be valid despite its age. It's very discouraging and demoralizing when stalebot comments on a PR that has been otherwise seemingly ignored...

**Comment by gilgongo** [comment]
I'm new to the Mastodon project, but from what I can tell there aren't enough maintainers for the amount of attention Mastodon is getting (judging by the commit logs). There's a similar comment to yours in the documentation repo too. And I notice there are currently 2,300 open issues (plus a large number of posts on the GH Discussions). That's not a good sign :-(

**Reply by Gargron** [reply]
Unlike many projects on GitHub, Mastodon is a user-facing project (not a library, not a tool for developers). The number of open issues cannot be compared to a typical GitHub software project. A lot of them are troubleshooting questions and feature requests, not bugs, and their amount or nature does not directly correlate with the very active development of this software.

**Reply by gilgongo** [reply]
Apologies, I didn't mean to imply the issues were bugs. Along with what @trwnh notes to do with PRs, what I meant was that a large number of open/unaddressed issues (both in this, the documentation repo and GH Discussion threads) makes it hard for anyone wanting to help out to get a feel for what the maintainers want to do and what's generally going on.  For example, https://github.com/mastodon/mastodon/issues/19528 doesn't look like it can be done any time soon. But that's just one Issue. 

Ideally I think feature requests, support and related issues would first be posted in GH Discussions. Any resolutions from those discussions could then be entered/tagged as either backlog Issues or PRs. That way, any Issue or PR would have been at least generally discussed and "triaged" so that newcomers would be able to see what the project's direction was.  

I know GitHub currently allows anyone to post an issue or a discussion, so the above is hard to do.

**Comment by Gargron** [comment]
FYI stalebot is gone, it was annoying me too.
